test/cpp/jit/CMakeLists.txt

```diff
@@ -74,7 +78,7 @@ if(USE_SYSTEM_ONNX)
   target_link_libraries(test_jit PRIVATE onnx_proto onnx)
 endif()
 
-set(JIT_TEST_DEPENDENCIES jitbackend_test gtest)
+set(JIT_TEST_DEPENDENCIES torch gtest jitbackend_test backend_with_compiler)
```

Comment:
torch wasn't needed before?

Reply:
torch is included into jitbackend_test and backend_with_compiler, so not necessarily needed here. It's just to explicitly show the dependency.